### DISCUSSION TITLE
无

### DISCUSSION THREAD

**Discussion by laqoome** [AlistGo/alist#2537] [Ideas]
无

**Comment by pipinian** [comment]
虽然不能挂载telegram这个无限存储，但如果能挂载小龙云盘也不错，听官方说数据不会丢失不用怕的

**Comment by laqoome** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by aoaoemoji** [comment]
这个云盘用过了 一定情况下无法下载文件 还经常发生 且来路和官方嘛 自己看吧 我怕跑路

**Comment by aoaoemoji** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
